### ISSUE TITLE
Duplicate images, and more.

### ISSUE THREAD

**Issue by SheElf** [WordPress/gutenberg#22231]
**Describe the bug**
A clear and concise description of what the bug is. When I upload an image, I get two of them. The nofollow code is missing. When I manually add a nofollow code to an image, I cannot switch it back to HTML. The Dashboard menu is missing from the left when I write my content. Why? Put it back. 

**To reproduce**

**Expected behavior**

**Screenshots**
If applicable, add screenshots to help explain your problem. Added.

**Editor version (please complete the following information):**
- WordPress version: 5.4.1 
- Does the website has Gutenberg plugin installed, or is it using the block editor that comes by default? [e.g: "gutenberg plugin", "default"]
- If the Gutenberg plugin is installed, which version is it? [e.g., 7.6]
![duplicate](https://user-images.githubusercontent.com/17300848/81425305-91d12e00-9114-11ea-87c3-4386e98f42c1.PNG)

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**
 - Browser  Chrome
 - Version newest version of Chrome

**Additional context**
- To report a security issue, please visit the WordPress HackerOne program: https://hackerone.com/wordpress.

**Comment by talldan** [comment]
@SheElf Thanks for getting in touch.

This sounds like it may be caused by a plugin you have active, as there would definitely be lots of reports of this issue if it happened in core WordPress. 

Please could you fill out the 'Steps to reproduce the behavior' section of the description. There are a few ways to upload images in WordPress, so it'd be great to know more about how to reproduce your issue.

**Comment by SheElf** [comment]
![duplicate images](https://user-images.githubusercontent.com/17300848/81826540-2234b780-94f5-11ea-9a1b-d1276f7327b0.PNG)

**Comment by SheElf** [comment]
I simply upload an image and sometimes I get duplicates as shown above. The second and third images are the same and it was uploaded only once. I'm using Chrome and uploading from my downloads.

**Comment by paaljoachim** [comment]
Hi @SheElf 

Can you test the following.
Turn off all plugins except the Gutenberg plugin.
Switch to the Twenty Twenty theme.

Then try again. If this still happens do also fill out how you are uploading the images. If you are dragging and dropping an image or browsing for an image etc.

Thanks.

**Comment by SheElf** [comment]
I know how to turn off all my plugins, but I don't have the 20/20 theme.   I
just uploaded another photo and it did the same thing. I two of the same
images.

**Comment by talldan** [comment]
> I simply upload an image and sometimes I get duplicates as shown above. The second and third images are the same and it was uploaded only once. I'm using Chrome and uploading from my downloads.

@SheElf This seems like an issue that you should take to the WordPress forums (https://wordpress.org/support/forums/).

The github issues here are for bugs related to the Block Editor. This doesn't seem to be a bug though, it's likely caused by the set up of your site (a configuration issue or a bug in a plugin).

**Comment by SheElf** [comment]
I'm unable to resolve this issue even with the plugins deactivated.

**Comment by paaljoachim** [comment]
Did you try switching to another theme?
This is really though not a Gutenberg issue. As Dan mentioned create a new issue in the https://wordpress.org/support/forums/ as others will give you some additional feedback to what you can test.

**Comment by SheElf** [comment]
I tried switching to twenty twenty and it totally messed up my blog. I went
back to my theme and now things are missing... like my copyright info and I
cannot get it back on. I'm NOT a tech person. Regardless, the duplication
persists.

**Comment by paaljoachim** [comment]
Contact the developer of the theme and mention some of the steps you have done, and what happened. He/She should hopefully be able to help. What is the url of your site?

**Comment by SheElf** [comment]
Yes, I did reach out to them. Hopefully they can offer some guidance. My
blog is Pausitive Living <https://pausitiveliving.ca>.

**Comment by SheElf** [comment]
I also wanted to mention that when I put images on my blog, the rel="noreferrer noopener no longer shows up.

**Comment by paaljoachim** [comment]
Good luck!

**Comment by SheElf** [comment]
Well, turns out the people from my theme won't help me. I wish I never
listened to you and tried a different theme. I'm not a tech person, so I'm
hooped.

**Comment by paaljoachim** [comment]
I am contacting you directly through e-mail and I will see what I can do to help you with this.

**Comment by SheElf** [comment]
Thank you.
